lib/Metadata.js

```diff
@@ -70,4 +70,30 @@ export default class Metadata {
 
     return response
   }
+
+  /**
+   * 
+   * @param {Number} id 
+   * @param {Object} userInfo 
+   */
+  async fetchMetadata(id, userInfo, token) {
```

Comment:
@steveoni This is a duplicate function. Just call the fetch method from Metastore: https://github.com/datopian/gift-portal/blob/e46611c02c686e46669bbc41cfd4662bc6feb96c/lib/Metastore.js#L38. It returns the metadata, including the id